### PR TITLE
Re-organise sidebar of contracts section + improve LSP25 standard page + improve type of executions in LSP6 standard page + improve the Contracts section for LSP6

### PR DESCRIPTION
**Pull request by CJ42**
# Contracts section sidebar

## before

image

## after

image

# LSP6 section for Types of Interactions

image

# LSP25 page improved

Moved the two sections below out of the LSP6 standard page and inside the LSP25 standard page:
- [How to sign relay transactions?](https://docs.lukso.tech/standards/access-control/lsp6-key-manager#how-to-sign-relay-transactions)
- [Out of order execution](https://docs.lukso.tech/standards/access-control/lsp6-key-manager#out-of-order-execution)

image

### CODE REVIEW

File: docs/standards/access-control/lsp6-key-manager.md [hunk 1]
```diff
@@ -800,120 +797,61 @@ As a result, this provide context for the Dapp on which data they can operate on
...
+| **Relay execution** <br/> _via Key Manager_                                                                        | The end user (Alice) signs a **payload** using the LSP25 signature format. <br/> A relayer (Bob) dispatch the transaction for the user to the Key Manager via [`executeRelayCall(...)`](../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#executerelaycall).                                                                                                                                          | Caller = relayer (Bob). <br/> The signer is the end user (Alice). <br/><span style={{color: "green"}}>**👍🏻 Alice does not pay for the gas fees (this method allows Alice to offload her gas fees to Bob).**</span>                                                                                                                                                                                                             |
```
Comment (JeneaVranceanu) [inline docs/standards/access-control/lsp6-key-manager.md:817]:
```suggestion
| **Relay execution** <br/> _via Key Manager_                                                                        | The end user (Alice) signs a **payload** using the LSP25 signature format. <br/> A relayer (Bob) dispatches the transaction for the user to the Key Manager via [`executeRelayCall(...)`](../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#executerelaycall).                                                                                                                                          | Caller = relayer (Bob). <br/> The signer is the end user (Alice). <br/><span style={{color: "green"}}>**👍🏻 Alice does not pay for the gas fees (this method allows Alice to offload her gas fees to Bob).**</span>                                                                                                                                                                                                             |
```

**Review by JeneaVranceanu — APPROVED** [review]
One typo 🙂

File: docs/contracts/overview/LSP17ContractExtension.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Extendable Contracts & Extensions Contracts
-sidebar_position: 4
+title: ⚙️ Extendable & Extensions Contracts
```
Comment (mustafademiray) [inline docs/contracts/overview/LSP17ContractExtension.md:2]: on the sidebar: Extendable & Extension**s** Contracts
on the page title: Extendable & Extension Contracts

**Comment by mustafademiray** [comment]
for the "Deployed Base Contracts" title, it always confuses me for a second if its something about Base(coinbase) chain. do we have an alternative for that? I am not sure how accurate but maybe something like: Deployed Main Contracts

**Review by mustafademiray — APPROVED** [review]

**Pull request merged**